src/journal/fuzz-journald-audit.c

```diff
@@ -9,7 +9,7 @@ int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size) {
 
         fuzz_setup_logging();
 
-        assert_se(manager_new(&m) >= 0);
+        assert_se(manager_new(&m, /* namespace */ NULL) >= 0);
```

Comment:
those argument prefix comments should be suffixed by `=`, i.e. `/* namespace= */` or so

here and everywhere else